narwhal/config/src/lib.rs

```diff
@@ -407,6 +407,8 @@ pub struct WorkerInfo {
     pub transactions: Multiaddr,
     /// Address to receive messages from other workers (WAN) and our primary.
     pub worker_address: Multiaddr,
+    /// Optional alternative address preferentially used by a primary to talk to its own workers.
```

Comment:
Yep. You could say something like "typically, this could be a LAN address only resolvable by the primary" to sketch what this is usable for, but I think most would get it.

Reply:
Thanks François, I think that's a good idea. No need for me to be overly opaque!